adal/build.gradle

```diff
@@ -70,13 +70,13 @@ android {
         debug {
             testCoverageEnabled false
             debuggable true
-            buildConfigField("String", "VERSION_NAME", "\"${versionName}\"")
+            buildConfigField("String", "VERSION_NAME", "\"${project.version}\"")
```

Comment:
Just a suggestion... maybe...

```suggestion
            buildConfigField("String", "VERSION_NAME", "\"${project.version}-debug\"")
```

Reply:
Good Idea!
I would take this as a separate work item, as I am not sure if this will break our existing pipelines or not.